src/fvtt_script.pyj

```diff
@@ -140,6 +140,16 @@ def rollInitiative(request, custom_roll_dice=""):
             addInitiativeToCombat(roll)
     )
 
+def popAvatar(request):
+    # Check for the beyond20 module, if it's there, we can pop the image.
+    module = game.modules.get("beyond20")

```

Comment:
Why beyond20 module? We don't need it. `ImagePopout` is a native FVTT API function.